### PR TITLE
[PSR-2] Fixes Part 1

### PR DESCRIPTION
**Pull request by gskema**
There are many warnings about PSR-2. This is part one (didn't have time to go through everything). Hopefully you can merge this :P

### CODE REVIEW

**Comment by julienbourdeau** [comment]
Hi,

I want to give your PR a deeper look but everything seems good ! Thank you.

File: classes/pdf/HTMLTemplateInvoice.php [hunk 1]
```diff
@@ -291,18 +291,18 @@ public function getContent()
          */
         $round_type = null;
         switch ($this->order->round_type) {
-        case Order::ROUND_TOTAL:
+            case Order::ROUND_TOTAL:
                 $round_type = 'total';
-            break;
-        case Order::ROUND_LINE;
+                break;
+            case Order::ROUND_LINE:
```
Comment (gskema) [inline classes/pdf/HTMLTemplateInvoice.php]: Somone typed semi colon here, they probably meant regular colon? I'm sorry I mixed this in, probably should've made a separate PR

**Comment by julienbourdeau** [comment]
Like we said on another PR, this need to be on develop branch.
I assume all your new PRs match this one.

**Comment by gskema** [comment]
Yes, shelved, then unshelved and split up into smaller PRs. Exluded some PHPDoc changes and the semicolon problem noted above. There is a separate PR to 1.6.1.x to fix this semi-colon

**Pull request closed without merging**